app/assets/javascripts/config/angularInitialize.js.coffee

```diff
@@ -136,6 +136,9 @@
             onConfirm: ->
               # fires only if user clicks 'ok' to leave page
               # reloads this stateChangeStart method with skipConfirm true
+              AnalyticsService.trackApplicationAbandon(ShortFormApplicationService.listing.listingID, null, "Leaving for " + toState.name)
+              $window.removeEventListener('unload', $rootScope.onUnload)
+
```

Comment:
question: When leaving an application, we show either the Angular modal or the browser dialog. We seem to be showing the browser dialog in most cases now, because users are navigating to a non-Angular page. Does this cover the browser dialog scenario?
![Screenshot 2024-12-17 at 3 36 19 PM](https://github.com/user-attachments/assets/f155c4be-2fe0-47a8-ba7b-8aeb09859818)
![Screenshot 2024-12-17 at 3 38 09 PM](https://github.com/user-attachments/assets/7d82b176-01c6-4787-bf46-a1c571c185fa)

Reply:
Yes! The browser dialog situation is covered by the onUnload handler that I added